app/web/features/search/FilterDialog.tsx

```diff
@@ -229,15 +258,47 @@ export default function FilterDialog({
               />
 
               <Select
-                id="can_host_status_filter"
-                value={hostingStatusFilter}
-                onChange={(e) => {
-                  setHostingStatusFilter(e.target.value as number);
-                }}
+                id="can_host_status_filter_1"
+                variant="outlined"
                 label={t("search:form.host_filters.hosting_status_field_label")}
+                multiple={true}
+                menuItems={true}
+                value={hostingStatusFilter}
+                onChange={(e) =>
+                  setHostingStatusFilter(
+                    e.target.value as TypeHostingStatusOptions
+                  )
+                }
+                input={
+                  <Input
+                    className={classes.inputHostingStatus}
+                    id="select-multiple-chip"
+                  />
+                }
+                className={classes.marginBottom}
+                native={false}
+                renderValue={(selected) => (
+                  <div className={classes.chips}>
+                    {(selected as TypeHostingStatusOptions).map(
```

Comment:
Could you leave a comment here that the type coercion is because the selected value type is unknown in the material ui Select component?